notNotes/schema.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from graphene_django import DjangoObjectType
+import graphene
+from .models import NotNote
+
+class NotNoteType(DjangoObjectType):
+
+    class Meta:
+        model = NotNote
+
+        interfaces = (graphene.relay.Node,)
+
+class Query(graphene.ObjectType):
+
+    notNotes = graphene.List(NotNoteType)
+
+    def resolve_notNotes(self, info):
+        return NotNote.objects.all()
+
+schema = graphene.Schema(query=Query)
```

Comment:
Great job with your GraphQL schema! It's good to know how to do things from the command line, but this interface makes things easier.